lib/mongo_ecto/normalized_query.ex

```diff
@@ -368,6 +368,7 @@ defmodule Mongo.Ecto.NormalizedQuery do
 
   defp mapped_pair_or_value({op, _, _} = tuple, params, pk, query, place) when is_op(op) do
     [pair(tuple, params, pk, query, place)]
+    |> :lists.flatten()
```

Comment:
Unfortunately not. I get `** (Mongo.Error) Can't canonicalize query: BadValue: $or must be an array 17287` when doing a simple `or` query. However, I forgot about `List.wrap/1`, which does appear to work.